src/test/java/subway/line/LineTest.java

```diff
@@ -0,0 +1,214 @@
+import subway.line.LineResponse;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("지하철노선 관련 기능")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+public class LineTest {
+    /**
+     * When 지하철노선을 생성하면
+     * Then 지하철 노선 목록 조회 시 생성한 노선을 찾을 수 있다
+     */
+    @DisplayName("지하철노선 생성")
+    @Test
+    void createSubwayLine() {
+        //When 지하철노선을 생성하면
+        LineRequest Line1 = new LineRequest("1호선", "blue", 1L,2L,10L);
+        createSubwayLine(Line1);
+
+        //Then 지하철 노선 목록 조회 시 생성한 노선을 찾을 수 있다
+        assertThat(getAllSubwayLines()).containsAnyOf(Line1.getName());
+    }
+
+    /**
+     * Given 2개의 지하철 노선을 생성하고
+     * When 지하철 노선 목록을 조회하면
+     * Then 지하철 노선 목록 조회 시 2개의 노선을 조회할 수 있다.
+     */
+    @DisplayName("지하철노선 목록 조회")
+    @Test
+    void getSubwayLineWithCreatedSubwayLines() {
+
+        // 데이터 무결성을 위함.
+        deleteAllSubway();
+
+        // Given 2개의 지하철 노선을 생성하고
+        LineRequest Line1 = new LineRequest("1호선", "blue", 1L,2L,10L);
+        createSubwayLine(Line1);
+        LineRequest Line5 = new LineRequest("5호선", "purple", 3L, 4L, 20L);
+        createSubwayLine(Line5);
+
+        // When 지하철 노선 목록을 조회하면 // Then 지하철 노선 목록 조회 시 2개의 노선을 조회할 수 있다.
+        assertThat(getAllSubwayLines()).hasSize(2);
+    }
+
+    /**
+     * Given 지하철 노선을 생성하고
+     * When 생성한 지하철 노선을 조회하면
+     * Then 생성한 지하철 노선의 정보를 응답받을 수 있다.
+     */
+    @DisplayName("지하철노선 조회")
+    @Test
+    void getStationLine() {
+
+        // Given 지하철 노선을 생성하고
+        LineRequest Line8 = new LineRequest("8호선", "pink", 6L,10L,200L);
+        Long lineId = createSubwayLine(Line8);
+
+        // When 생성한 지하철 노선을 조회하면
+        LineResponse lineResponse = getSubwayLine(lineId);
+
+        // Then 생성한 지하철 노선의 정보를 응답받을 수 있다.
+        assertThat(Line8.getName()).isEqualTo(lineResponse.getName());
+        assertThat(Line8.getColor()).isEqualTo(lineResponse.getColor());
+        assertThat(Line8.getUpStationId()).isEqualTo(lineResponse.getUpStationId());
+        assertThat(Line8.getDownStationId()).isEqualTo(lineResponse.getDownStationId());
+        assertThat(Line8.getDistance()).isEqualTo(lineResponse.getDistance());
+    }
+
+    /**
+     * Given 지하철 노선을 생성하고
+     * When 생성한 지하철 노선을 수정하면
+     * Then 해당 지하철 노선 정보는 수정된다
+     */
+    @DisplayName("지하철노선 수정")
+    @Test
+    void updateSubwayLine() {
+
+        // Given 지하철 노선을 생성하고
+        LineRequest Line8 = new LineRequest("8호선", "pink", 6L,10L,200L);
+        Long lineId = createSubwayLine(Line8);
+
+        // When 생성한 지하철 노선을 수정하면
+        LineRequest updateLine8 = new LineRequest(Line8.getName(), "black", Line8.getUpStationId(),4L,100L);
+        LineResponse lineResponse = updateSubwayLine(lineId, updateLine8);
+
+        // Then 해당 지하철 노선 정보는 수정된다
+        assertThat(updateLine8.getName()).isEqualTo(lineResponse.getName());
```

Comment:
```suggestion
        assertThat(lineResponse.getName()).isEqualTo(updateLine8.getName());
```
assertThat메소드의 파라미터로 검증하려는 값을, isEqualTo의 파라미터로 기대하는 값을 작성해보시면 어떨까요? 실패시에 메세지도 명확하게 확인할 수 있어요 !